### PR TITLE
fix and simplify project update analytics tracker

### LINKED ISSUE #86: Add back in missing values from 'Project Created' Track call for Segment
## 📋 Anything to add?

current track called calls the properties from an object called formattedProject in projectResolver.ts and returns to segment -
```
analytics.track({
  userId: 'givethId-67',
  event: 'Project created',
  properties: {
    admin: 'atvatvaev',
    categories: [
      'non-profit'
    ],
    description: '<p>atrythrybrah</p>',
    imageStatic: '2',
    impactLocation: '',
    organisationId: 1,
    title: 'arfvadv',
    walletAddress: '0x38b3f21c23424F955EC5D259b04eE198862d6847'
  }
});
```

this is problematic because we're missing some important fields in order for autopilot and segment to work properly
we need to be capturing at least:

- project owner's email
- project owner's name
- project slug
- project title
- project owner user ID
- project wallet address.

### LINKED ISSUE #94: fix track calls for project update - donor
currently there are two analytic.track calls being sent to segment for when a project makes an update on giveth.io  from the giveth impact-graph. The intention is to eventually send emails to donors letting them know a project they donated to has an update. 

one is for the project owner
and another for donors to the project. the problem is that a track call is being sent for every instance of donation made not for every unique address that made a donation. 

this is problematic in that if a user has made several donations to a single project and the project has an update they would receive one email for each donation they had made to the project. meaning for one project update a user could receive several redundant emails

from projectresolver.ts 

```
  const donations = await this.donationRepository.find({
      where: { project: { id: project.id } },
      relations: ['user']
    })

    donations.forEach(donation => {
      analytics.track(
        'Project updated - donor',
        `givethId-${donation.user.id}`,
        {
          project,
          update: title,
          donation
        },
        null
      )
```

### PR DESCRIPTION
**Pull request by divine-comedian**
updated the analytics tracker for project update, related to issue #94 

also trimmed down the amount of info included in the analytics.

### CODE REVIEW

File: resolvers/projectResolver.ts [hunk 1]
```diff
@@ -615,14 +615,18 @@ export class ProjectResolver {
       createdAt: new Date(),
       isMain: false
     })
-
+    const projectUpdateInfo = {
+      title: project.title,
+      email: project.users[0].email,
```
Comment (cquinterom096) [inline resolvers/projectResolver.ts]: I'll do a small change here @divine-comedian, project.users[0].email will break for old projects.

**Review by cquinterom096 — CHANGES_REQUESTED** [review]

File: resolvers/projectResolver.ts (hunk 1 above)
Comment (divine-comedian) [reply resolvers/projectResolver.ts]: okay, what part needs to be changed?

Comment (cquinterom096) [reply resolvers/projectResolver.ts]: The user object is already the owner. So typing user.email should do it.

Comment (divine-comedian) [reply resolvers/projectResolver.ts]: so when I log user.email on  projectUpdate it returns undefined
however logging project.users[0].email returns the correct info

**Review by cquinterom096 — APPROVED** [review]
looks good!

**Comment by MoeNick** [comment]
@divine-comedian According to our meeting, pls request all backed code reviews from @aminlatifi 
thanks

**Pull request merged**